consensus/istanbul/uptime/uptime_test.go

```diff
@@ -0,0 +1,124 @@
+package uptime
+
+import (
+	"math/big"
+	"reflect"
+	"testing"
+)
+
+func TestGetUptimeMonitoringWindow(t *testing.T) {
+	type args struct {
+		epochNumber        uint64
+		epochSize          uint64
+		lookbackWindowSize uint64
+	}
+	tests := []struct {
+		name  string
+		args  args
+		want  uint64
+		want1 uint64
+	}{
+		{"tally on first epoch", args{1, 10, 2}, 1 + 2, 9},
```

Comment:
Replace "tally" with "monitoringWindow" or something similar.

Also, I don't like the `1 + 2` and `11 + 2`, instead of `3` and `13`.  Or at least it should be consistent and instead of `9` and `19` is should be `10 - 1` and `20 - 1`) (though I don't like that either).